static/src/stylesheets/base/_inline-svgs.scss

```diff
@@ -6,6 +6,15 @@
     }
 }
 
+.inline-icon.light-grey {
+    fill: colour(neutral-3);
+}
+
+.inline-close.small {
```

Comment:
`.inline-icon--light-grey` would be a typical BEM selector, you wouldn't normally chain the selector under BEM conventions.